pkg/actors/actors.go

```diff
@@ -944,7 +949,7 @@ func (a *actorsRuntime) startReminder(reminder *reminders.Reminder, stopChannel
 			}
 
 			err = a.executeReminder(reminder, false)
-			diag.DefaultMonitoring.ActorReminderFired(reminder.ActorType, err == nil)
+			diag.DefaultMonitoring.ActorReminderFired(reminder.ActorType, metricsSuccessStatusString[err == nil])
 			if err != nil {
```

Comment:
@filintod Why is this changing for this metric alone? Aren't we using `er r== nil` for all other metrics? 
For consistency I say we keep `err == nil` itself. And if needed create a separate issue to change all metrics to have the success, failure change.

Reply:
ok, I see, they use `strconv.FormatBool` later. thanks